tools/cldr-code/src/main/java/org/unicode/cldr/util/TimeDiff.java

```diff
@@ -0,0 +1,75 @@
+package org.unicode.cldr.util;
+
+import com.ibm.icu.text.MeasureFormat;
+import com.ibm.icu.text.MeasureFormat.FormatWidth;
+import com.ibm.icu.text.RelativeDateTimeFormatter;
+import com.ibm.icu.util.Measure;
+import com.ibm.icu.util.MeasureUnit;
+import com.ibm.icu.util.ULocale;
+import java.util.Locale;
+
+public class TimeDiff {
+    public static String timeDiff(long a) {
+        return timeDiff(a, System.currentTimeMillis());
+    }
+
+    public static String durationDiff(long a) {
+        return timeDiff(System.currentTimeMillis() - a);
+    }
+
+    private static final long ONE_SECOND = 1 * 1000;
+    private static final long ONE_MINUTE = 60 * 1000;
+    private static final long ONE_HOUR = 3600 * 1000;
+    private static final long ONE_DAY = 86400 * 1000;
+    private static final double ONE_YEAR = ONE_DAY * 364.25;
+
+    /**
+     * @returns string representation of the difference between the two millisecond values
+     */
+    public static String timeDiff(long a, long b) {
```

Comment:
I'm surprised there's not a standard library for this

Reply:
Icu has the formatting but not the field resolution.

Reply:
86400 = Duration.ofDays(1).getSeconds()

I guess there would be a tiny performance penalty

Reply:
This makes it very hard to import data into spreadsheets. You really don't want to force people to write functions to convert various measures of time into real numbers that can be compared.

For now, it should be just a number of days, eg 3.2 days — and without the "days" string; that should be in the column header.

Reply:
This is for ephemeral information such as the vetting viewer page load, not downloaded data. Last seen in the spreadsheet uses a UTC timestamp.

Reply:
ok, good